services/QuillLMS/app/controllers/cms/district_admins_controller.rb

```diff
@@ -38,4 +34,39 @@ def set_district_admin
   def district_admin_params
     params.require(:email)
   end
+
+  private def create_district_admin_user_for_existing_user(user, new_user: false)
+
+    if @district.district_admins.find_by(user_id: user.id)
```

Comment:
Opt for `exists?` instead of `find_by` if the object isn't needed:

`@district.district_admins.exists?(user_id: user.id)`

Reply:
good call!